Dynatrace/dynatrace-operator
# Remove version from `matchLabels` in Helm and Activegate

# Description
No label that changes overtime should be part of the matchLabels as those are immutable. Also reintroduces the old labels to be only used as matchLabels for know, will be replaced in the future

## How can this be tested?
After deploy, check if the deployments/daemonset/statefulsets have all the labels (which has the version, feature etc)
but the matchLabels should only have things that don't change (component, name)

## Checklist
